EventsExpress/ClientApp/src/components/profile/editProfile/editBirthday.js

```diff
@@ -33,7 +34,7 @@ const validate = values => {
     return errors;
 };
 
-const EditBirthday = ({ handleSubmit, pristine, reset, submitting, error }) => {
+const EditBirthday = ({ handleSubmit, pristine, reset, submitting, error, onClose }) => {
```

Comment:
All of the "Edit..." components share similar structure. Let's extract similar functionality into a reusable component.